packages/scripts/src/schema-flatten/config.ts

```diff
@@ -0,0 +1,48 @@
+/**
+ * Ignore any collisions based on the keys in the bootstrap package
+ */
+export const collisionIgnoreMap = {
```

Comment:
Might need collision ignores for those extra bootstrap env vars I mentioned.

Reply:
Yeah, ill make this scripted